app/views/home/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="card-columns">
+<!-- <div class="card-columns">
```

Comment:
It's always a good idea to not merge unusable code (commented code) into the mainline. 🗑